src/styles/block/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  justify-content: center;
+  align-items: center;
```

Comment:
Properties 'justify-content' and 'align-items' are used for flex or grid containers. If '.card' is not a flex or grid container, these properties will have no effect.